Removes guns and viruses from maint loot spawners

Kurfursten opened the pull request:
Removes some highly dangerous stuff from maint loot spawner and moved it to its own spawner, including:
- Guns
- Powersink
- Viruses

However, to not be anti-fun, some new entertaining stuff has been added to make up for it. _Italicized if normally unobtainable._
- Syndicate wrapping paper
- Suspicious toolbox
- Pop out cake
- _Multi-person wheelchair_
- _Headset earmuffs_
- _Peanut_
- _Box of wind grenades_
- Box of smoke grenades
- Box of metal foam grenades
- Pre-loaded soldering tool
- _Swiss army knife_

I think this is a fair tradeoff to make Deff a little less crazy without sacrificing the fun of a great discovery in maintenance. Did you know Powersink was listed twice on the spawner? That explains why we saw so many.

clusterfack commented:
We should make the Swiss army knife an item vox traders can buy from the galactic market and trade to the crew for shinies.

clusterfack commented:
I'm interested in what people think about this though

unid15 commented:
>Multi-person wheelchair

#5901

Kurfursten commented:
Most of the stuff from that report, like untaseable while on the chair, doesn't apply anymore, right?

unid15 commented:
Right

FudgePucker commented:
I think an intergalactic market for vox would be a good idea.

It would make trading more viable if they had shit not avalible anywhere else. Maybe things like gokarts

ihadtoregisterforthis commented:
Other than the wheelchair sounds alright.

Exxion commented:
Why is the wheelchair bad?
It's harmless on its own

DrCelt commented:
>Multi-person wheelchair
No.

Exxion commented:
But why not?

DrCelt commented:
muh immulshions

Exxion commented:
Fuck your immulsions
Just make it so that each new person buckled in sits slightly higher than the last and mention in the description that it has a lot of buckles

DrCelt commented:
that is not how gravity works, fuck you.

Exxion commented:
We have a gravity generator
I don't think anyone cares how gravity actually works

DrCelt commented:
@Exxion your argument is nonsense and the multi-wheelchair is a meme non-item

Exxion commented:
It's definitely a meme non-item but fuck you my argument is flawless

DrCelt commented:
How about, I don't think meme non-items should be obtainable that isn't adminspawn only, then.

Exxion commented:
skub

Technically an item but still a meme

DrCelt commented:
@Exxion just because X the meme is in the game already doesn't mean Y the meme should be.
Also I didn't know you could acquire skub.

Exxion commented:
It's a gashapon toy
And the memechair tower is hilarious

DrCelt commented:
@Exxion Hilarity is subjective though

Exxion commented:
Doesn't make it not hilarious

NigglyWiggly commented:
>Peanut

What


>Celt being a no-fun shit
:eggplant:

Kurfursten commented:
@NigglyWiggly Peanuts are a secret crop that exists in game but aren't available at the vendor. They could use a tiny bit of work but they're functional.

Most people would just eat/ignore the peanut instead of seed extracting it anyway.

NigglyWiggly commented:
Why are they secret, though?

Kurfursten commented:
There are a lot of crops we once had that were removed.

I'm *guessing* the reason they were taken out is because some idiot named the plant "peanut vines" because they didn't know dick about how peanuts grow. It's just cosmetic.

clusterfack commented:
What a moron everyone knows peanuts grow on trees

NigglyWiggly commented:
That's just an old rumor, heard it through the nut vine, y'know?

Intigracy commented:
I'm glad celt isn't a maintainer.

Intigracy commented:
Also @clusterfack I think the vox should get a special autolathe to make special items, like the vox dart gun.

clusterfack commented:
Nah they should have something like a cargo supply console that with a market that varies in price for things from exotic places or the vox (homeworld? clan place? whatever)

Kurfursten commented:
The problem with that is adding a bunch of interesting stuff that people want to buy.

clusterfack commented:
Nah thats quitter talk

gbasood commented:
We already have a bunch of shit people pick up around the station. What makes you think you couldn't add purchasable shit?
Or worst case add some of it to merch console

Intigracy commented:
eww the merch console
let's get rid of it and give it all to vox traders

SonixApache commented:
fuck you duny

gbasood commented:
\>open for 3 days
\>everyone wants it
HAHAHHAHAH 
# MEEERGE

Megamaw commented:
>not atomic
Nice

Intigracy commented:
uh he balanced loot spawners how is this not atomic

Kurfursten commented:
@Intigracy I think he's making a joke about how I also corrected a misleading line in the headset earmuff description. It used to say that it had a radio key for engineering and service but that was taken out, it doesn't have those keys anymore, so I fixed the description in this PR.

So memes
